Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -53,6 +56,64 @@ theorem sin_lt {x : ℝ} (h : 0 < x) : sin x < x := by
   norm_num
 #align real.sin_lt Real.sin_lt
 
+lemma cos_quadratic_lower_bound : 1 - x ^ 2 / 2 ≤ cos x := by
```

Comment:
I'd rather have a lemma name that corresponds more closely to the statement.

Reply:
I'm not sure symbol-reading is better here. After all, this is the best quadratic lower bound you can get on `cos`, and something like `one_sub_sq_div_two_le_cos` does not really get that message across. Also please note the existing `sin_bound` and `cos_bound`.

Reply:
>  this is the best quadratic lower bound you can get

I'd argue that `cos_quadratic_lower_bound` is more suited to the theorem that actually states this; the current version just gives *a* lower bound, without proving it is tight.

Reply:
I'm not sure how such a statement would look like, but certainly I wouldn't call it `cos_quadratic_lower_bound` because there will be more specific symbols to refer to.

Reply:
`alias`?

Reply:
What do you mean, Yury?

Reply:
I guess that Yury means: name the theorem according to the naming convention, and create an alias with the name that you currently use.

Reply:
Might as well rename the lemma at this point